joomla/joomla-cms
# [4.0] View Private message: Add separator colon : while previewing the message #pbf19 #bug #26915

…he message

Pull Request for Issue # .

### Summary of Changes
Add seperator like colon while viewing message


### Testing Instructions
1. Open any private message
2. Verify that the label has a seperator after it


### Expected result
There should be a seperator after the label


### Actual result
Colon is shown after the label so that the user can get a better view


### Documentation Changes Required
